### PR TITLE
improves/clarifies timecode sections

### PR DESCRIPTION
**Pull request by ablwr**
Version two. :)

### CODE REVIEW

**Comment by ablwr** [comment]
Whatcha think? 😇

File: notes.md [hunk 1]
```diff
@@ -159,81 +160,81 @@ The StereoMode used to be part of Matroska v2 but it didn't meet the requirement
...
 
-If a Cluster's Timecode is set to zero, it is possible to have Blocks with a negative Raw Timecode. Blocks with a negative Raw Timecode are not valid.
+If a `Cluster`'s `Timecode Element` is set to zero, it is possible to have `Block Elements` with a negative `Raw Timecode`. `Block Elements` with a negative `Raw Timecode` are not valid.
 
 ## Raw Timecode
 
-The exact time of an object represented in nanoseconds. To find out a Block's Raw Timecode, you need the Block's timecode, the Cluster's Timecode, and the TimecodeScale. For calculation, please see the see the TimecodeScale notes.
+The exact time of an object SHOULD be represented in nanoseconds. To find out a `Block`'s `Raw Timecode`, you need the `Block`'s `Timecode Element`, the `Cluster`'s `Timecode Element`, and the `TimecodeScale Element`. For calculation, please see the see the [`TimecodeScale` notes](#timecodescale).
```
Comment (retokromer) [inline notes.md]: >For calculation, please see

This sounds strange to me, because it’s actually the following ¶

File: notes.md [hunk 2]
```diff
@@ -159,81 +160,81 @@ The StereoMode used to be part of Matroska v2 but it didn't meet the requirement
...
 
-The next step is to calculate the Absolute Timecode - that is the timecode that will be stored in the Matroska file. Here the application has to divide the Raw Timecode from the previous paragraph by the TimecodeScale factor and round the result: `AbsoluteTimecode = round(RawTimecode / TimecodeScale_facotr)` which will result in the following for our example: `AbsoluteTimecode = round(1186848073 / 22675) = round(52341.7011245866) = 52342`. This number is the one the application has to write to the file.
+The next step is to calculate the Absolute Timecode - that is the timecode that will be stored in the Matroska file. Here the application has to divide the Raw Timecode from the previous paragraph by the `TimecodeScale` factor and round the result: `Absolute Timecode = round(Raw Timecode / TimecodeScale_factor)` which will result in the following for our example: `Absolute Timecode = round(1186848073 / 22675) = round(52341.7011245866) = 52342`. This number is the one the application has to write to the file.
```
Comment (retokromer) [inline notes.md]: \`Absolute Timecode\` and \`Raw Timecode\`, I guess

File: notes.md [hunk 3]
```diff
@@ -159,81 +160,81 @@ The StereoMode used to be part of Matroska v2 but it didn't meet the requirement
...
 
-Now our file is complete, and we want to play it back with another application. Its task is to find out which sample the first application wrote into the file. So it starts reading the Matroska file and finds the TimecodeScale factor 22675 and the audio sample rate 44100. Later it finds a data block with the Absolute Timecode of 52342. But how does it get the sample number from these numbers?
+Now our file is complete, and we want to play it back with another application. Its task is to find out which sample the first application wrote into the file. So it starts reading the Matroska file and finds the `TimecodeScale` factor 22675 and the audio sample rate 44100. Later it finds a data block with the Absolute Timecode of 52342. But how does it get the sample number from these numbers?
```
Comment (retokromer) [inline notes.md]: \`Absolute Timecode\`

File: notes.md [hunk 4]
```diff
@@ -159,81 +160,81 @@ The StereoMode used to be part of Matroska v2 but it didn't meet the requirement
...
     Track 3: PAL Audio
 
-Because the NTSC track is at the original speed, it will used as the default value of 1.0 for its TrackTimecodeScale. The video will also be aligned to the NTSC track with the default value of 1.0.
+Because the NTSC track is at the original speed, it will used as the default value of 1.0 for its `TrackTimecodeScale`. The video will also be aligned to the NTSC track with the default value of 1.0.
 
-The TrackTimecodeScale value to use for the PAL track would be calculated by determining how much faster the PAL track is than the NTSC track. In this case, because we know the video for the NTSC audio is being played back at 24fps and the video for the PAL audio is being played back at 25fps, the calculation would be:
+The `TrackTimecodeScale` value to use for the PAL track would be calculated by determining how much faster the PAL track is than the NTSC track. In this case, because we know the video for the NTSC audio is being played back at 24fps and the video for the PAL audio is being played back at 25fps, the calculation would be:
 
-    (25 / 24) = ~ 1.04166666666666666667
+24/25 ≈ 1.04166666666666666667
```
Comment (retokromer) [inline notes.md]: 24/25 = 0.96

Comment (ablwr) [reply notes.md]: Oh hahahaahahhaa thank you

File: notes.md (hunk 1 above)
Comment (ablwr) [reply notes.md]: Just gonna drop the "see the TimecodeScale notes" sentence

**Comment by ablwr** [comment]
Done. Thank you @retokromer!

**Comment by robUx4** [comment]
LGTM.
It might be time to rename Timecode* to Timestamp*.

**Comment by ablwr** [comment]
@robUx4 How would we collectively come to that consensus?

**Comment by robUx4** [comment]
That should an issue discussed on CELLAR. But I don't think anyone will disagree. Since the XML contains the cppname, we can retain backward compatibility with the code.

**Comment by ablwr** [comment]
Thanks for making an issue about that. I am merging this!

**Pull request merged**